pkg/plugins/crypto/api.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import "github.com/google/tink/go/keyset"
+
+// Provider interface for Crypto ctx
+type Provider interface {
+}
+
+// Creator method to create new pluggable Crypto service
+type Creator func(prov Provider) (AriesCrypto, error)
+
+// AriesCrypto is an interface defining a pluggable Crypto operations provider to Aries agents.
+// It's only responsible for pure crypto operations and nothing more. Envelope building and keys management are managed
+// in other interfaces
+type AriesCrypto interface {
```

Comment:
The Aries prefix isn't needed.

Reply:
I wanted to differentiate it from the currently used Crypto interface. I will remove it..

Reply:
prefix removed